ddtrace/tracer/spancontext.go

```diff
@@ -181,12 +181,12 @@ func (c *spanContext) ForeachBaggageItem(handler func(k, v string) bool) {
 	}
 }
 
-func (c *spanContext) setSamplingPriority(p int, sampler samplernames.SamplerName) {
```

Comment:
Although I appreciate descriptive function names, I feel this one is too verbose. I would rather keep it as is or change it to a more general `setSampling` with a more detailed comment docs explaining what changes happen when calling to these functions.

This applies to other equally named functions. I worry that the name may turn irrelevant or not accurate in the future if we add any more side effects derived from context.

Reply:
Renamed it back.